docs/getting-started/linux-installation.md

```diff
@@ -26,66 +26,37 @@ ExecStart=/usr/bin/dockerd -H fd:// --dns=172.17.0.1
 ```
 1. Set up the docker0 network as trusted
     - `sudo firewall-cmd --zone=trusted --add-interface=docker0 && sudo firewall-cmd --zone=trusted --add-interface=docker0 --permanent`
-1. Start the docker daemon
-    - `sudo systemctl start docker`
+1. Restart the docker daemon
```

Comment:
Is there an earlier step which runs start?

Reply:
Well, folks may have it already started, restart will work even if the service is inactive or active. Made more sense, is it confusing?